src/toil/cwl/cwltoil.py

```diff
@@ -1926,6 +1926,12 @@ def main(args: Union[List[str]] = None, stdout: TextIO = sys.stdout) -> int:
         action="store_true",
         help="Enable loading and running development versions of CWL",
     )
+    parser.add_argument(
+        "--enable-ext",
+        action="store_true",
+        help="Enable loading and running 'cwltool:' extensions to the CWL standards.",
+        default=False,
+    )
```

Comment:
Oh, good! This will allow for InplaceUpdateRequirement before CWL v1.2.

It also allows for the rest of https://github.com/common-workflow-language/cwltool/blob/main/cwltool/extensions.yml & https://github.com/common-workflow-language/cwltool/blob/main/cwltool/extensions-v1.1.yml ; which by a quick glance should not be a problem.

However, maybe the tests for those should also be copied over into Toil?

Reply:
@mr-c That sounds like a good idea to me!  I think I prefer it in a separate PR.  I'll make an issue for it and follow up with it once MPI support is in.